.github/workflows/npm-run-package.yml

```diff
@@ -0,0 +1,46 @@
+name: 'npm run package'
+# Main use case: repackage when Dependabot updates a dependency
+on:
+  push:
+    branches:
+      - 'dependabot/**'
```

Comment:
should probably be `'dependabot/npm_and_yarn/**'` since we also get dependabot PRs for GitHub actions.

Reply:
Good point!

Reply:
Done.